### PR TITLE
docs(man): use two single quotes for describing options

### LINKED ISSUE #1435: No rule for creating html version of manpage
See #1433.

A makefile rule for creating `man/ctags.1.html` from `man/ctags.1.rst` is needed.
We can use rst2html for it, however some part is wrongly marked up. https://github.com/universal-ctags/ctags/pull/1433#issuecomment-305650424

### PR DESCRIPTION
**Pull request by masatake**
Restructured text can recognize options list in an
input file. However, the notation for the options list
is not enough flexible for describing all options of
ctags. As the result, the HTML output converted from
crags.1.rst by rst2html becomes ugly.

So in this chage, use definition list with
``--two-single-quotes`` to describe options
instead of using options list.

Partially close #1435.

Signed-off-by: Masatake YAMATO <yamato@redhat.com>

### CODE REVIEW

**Comment by masatake** [comment]
@k-takata, withi this change, I expect the output of rst2html becomes better. Could you verify this in your side?

File: man/ctags.1.rst.in [hunk 1]
```diff
@@ -714,7 +714,7 @@ built-in language names.
 --license
```
Comment (k-takata) [inline man/ctags.1.rst.in]: Should be surrounded by ``` `` ```.

**Comment by masatake** [comment]
Updated.

**Comment by k-takata** [comment]
This PR itself looks good to me.
However your commit message says "Close #1435", but it is not enough for closing it.
I wrote as following in #1435:
> A makefile rule for creating man/ctags.1.html from man/ctags.1.rst is needed.

**Comment by masatake** [comment]
Oh, sorry. I change "Partially close #1435".

About man/ctags.1.html, I wonder where I should write it to.
ctags.1 is available therefore I think we don't have to install ctags.1.html to the system where ctags is installed to. I assume ctags.1.html is an extra prodcut like files under docs directory. My idea is adding a Makefile to man/ directory putting rules for generating html files in the Makefile. Does this align what you tihnk?

**Comment by k-takata** [comment]
Agree. The html files are mostly for Windows zip package and no need to install on normal unix like systems by default.
(Maybe it's better if it is also available on http://docs.ctags.io/en/latest/ .)

**Comment by masatake** [comment]
> (Maybe it's better if it is also available on http://docs.ctags.io/en/latest/ .)
I agree. 
Generating .html files for the man page is MUST for releasing 6.0. However, making them available on the web page is optional. Let's focus on tasks for releasing:-).

**Pull request merged**